### PR TITLE
added poap nfts on xdai

### PR DESCRIPTION
**Pull request by poocart**
Per title.

### CODE REVIEW

File: src/services/poap.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+// types
+import type { Collectible } from 'models/Collectible';
+
+
+/* eslint-disable i18next/no-literal-string */
+const xDaiRpcUrl = 'https://dai.poa.network/';
+const poapContractAddress = '0x22c1f6050e56d2876009903609a2cc3fef83b415';
+const poapSubgraphName = 'poap-xyz/poap-xdai';
+/* eslint-enable i18next/no-literal-string */
```
Comment (IAmKio) [inline src/services/poap.js:44]: I know this is temporary before we move to the backend so i'm happy to keep this here for now

File: src/services/poap.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+  const tokensWithMetadata = await Promise.all(tokenIds.map(async (tokenId) => {
+    const tokenMetadataUri = await collectibleContract.tokenURI(tokenId).catch(() => null);
+    if (!tokenMetadataUri) return null;
+
+    try {
+      const { data } = await httpRequest.get(tokenMetadataUri);
+      return { ...data, tokenId };
+    } catch (error) {
+      //
```
Comment (IAmKio) [inline src/services/poap.js:79]: Shall we log and report something here?

**Review by IAmKio — CHANGES_REQUESTED** [review]
Just a small point re: error logging but apart from that - all good!

File: src/services/poap.js (hunk 1 above)
Comment (poocart) [reply src/services/poap.js:44]: Exactly ❤️  didn't want to pollute configs/constants

File: src/services/poap.js (hunk 2 above)
Comment (poocart) [reply src/services/poap.js]: Added

**Review by IAmKio — APPROVED** [review]

**Pull request merged**